Makefile

```diff
@@ -23,6 +23,7 @@ LINT_SOURCES = \
         test/cpp/asyncprogressworker.cpp \
         test/cpp/asyncworkererror.cpp \
         test/cpp/bufferworkerpersistent.cpp \
+	test/cpp/error.cpp \
```

Comment:
Indentation seems a bit off here.  Maybe that's because it's a tab instead of spaces like on the surrounding lines?